tests/controllers/__init__.py

```diff
@@ -8,3 +8,17 @@
 from .controller_with_throttling import ControllerWithThrottlingOnMethod
 from .no_authentication_controller import NoAuthenticationController
 from .simple_controller import SimpleController
+
+
+__all__ = (
```

Comment:
Module level "dunders" (i.e. names with two leading and two trailing underscores) such as __all__, __author__, __version__, etc. should be placed after the module docstring but **before** any import statements except from __future__ imports.

https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names

Reply:
Just deleted all it